src/Scripting/CSharpTest/CommandLineRunnerTests.cs

```diff
@@ -135,7 +135,7 @@ public void Await()
                 runner.Console.Error.ToString());
         }
 
-        [Fact]
+        [Fact(Skip="https://github.com/dotnet/roslyn/issues/17043")]
```

Comment:
That issue has no specific information about how this test is failing, at least it doesn't mention it explicitly. I am curious how exactly does this test fail and why are we disabling it instead of fixing it or limiting it to environments on which it is passing? BTW, what is CSI in this context?